scipy/fftpack/logarithmic.py

```diff
@@ -0,0 +1,256 @@
+"""
+FFTLog - fftlog.py
+"""
+# Created by Dieter Werthmüller, January 2017
+from __future__ import division, print_function, absolute_import
+
+import atexit
+import numpy as np
+from . import _fftl
+from .basic import _asfarray
+
+__all__ = ['fftlog', 'fftlogargs']
+
+atexit.register(_fftl.destroy_fftl_x_cache)
+atexit.register(_fftl.destroy_fftl_w_cache)
+del atexit
+
+
+def fftlog(x, dlogr, mu='sine', q=0.0, kr=1.0, rk=1.0):
```

Comment:
Are all these parameters really necessary, and if so can they be given more descriptive names? Especially ``kr`` and ``rk`` seem like parameters that are taken over from the Fortran code but are unintuitive for users.